src/debug/daccess/request.cpp

```diff
@@ -3726,7 +3726,7 @@ ClrDataAccess::GetSyncBlockData(unsigned int SBNumber, struct DacpSyncBlockData
             }
 #endif // FEATURE_COMINTEROP
 
-            pSyncBlockData->MonitorHeld = pBlock->m_Monitor.m_MonitorHeld;
+            pSyncBlockData->MonitorHeld = pBlock->m_Monitor.m_lockState.VolatileLoad().GetMonitorHeldState();
```

Comment:
I would like 'ProbablyMonitorHeld to be a property and then define that to be  m_lockState.VolatileLoad().GetMonitorHeldState().  In general I would like m_lockState to be private to AwareLock

Reply:
Yea I should have done that, will do